### DISCUSSION TITLE
Plans to support non AWS/GCP providers?

### DISCUSSION THREAD

**Discussion by BadgerOps** [opstrace/opstrace#138] [Q&A] [answered]
Hi there!

opstrace looks really great, but I do not currently utilize either AWS or GCP. Are there any plans to support other providers, like Azure or DigitalOcean? I'm happy to help test/document/architect things on the DO front specifically. 

Thank you!

-BadgerOps

**Comment by MatApple** [comment, marked as answer]
Hey BadgerOps! We'd love to support more than just AWS and GCP! We also have to think about the investment in testing infrastructure and the additional overhead that another provider may cause when adding new features. One of the critical aspects to the projects success will be in its automated testing and ability to evolve quickly - two important functions that we're striving for.

As such, we don't have near-term plans for expanding to more providers but we'd love to hear more about your use case, and we can go from there! We'd be more than happy to assist any effort to support Digital Ocean if it made sense for the community.

**Reply by BadgerOps** [reply]
Totally understand that viewpoint, its certainly a difficult thing to balance especially for a startup, which is something I'm in the middle of as well. I appreciate the quick response!
